gdx/src/com/badlogic/gdx/math/Intersector.java

```diff
@@ -151,7 +151,7 @@ public static boolean isPointInPolygon (float[] polygon, int offset, int count,
 	private final static Vector2 e = new Vector2();
 
 	/** Intersects two convex polygons with clockwise vertices and sets the overlap polygon resulting from the intersection.
```

Comment:
Doesn't it already specifies it here the polygons should be clockwise?

Reply:
XD  i read it 100 times previous month, and i never saw it.